apps/meteor/tests/end-to-end/api/01-users.js

```diff
@@ -140,6 +140,46 @@ describe('[Users]', function () {
 			await deleteUser(user);
 		});
 
+		it('should create new user after trimming faulty email', async () => {
+			await setCustomFields({ customFieldText });
```

Comment:
is this necessary?

Reply:
is the API test unnecessary? I added on the request of Hugo. Can remove if it's unnecessary.

Reply:
Ah alright. I kept this test similar to the one above it, since I thought it's standard practice. I do see what you mean though; I'll remove this line. Is everything else ok?